src/Parser/BalanceDruid/Modules/Spells/FullMoon.js

```diff
@@ -40,14 +42,14 @@ class FullMoon extends Module {
     const offSet = this.firstMoonTime + 15;
     const totalFromCD = ((this.owner.fightDuration / 1000) - offSet) / 15;
     const eachMoon = Math.floor(totalFromCD / 3);
-    let fmAvailableCasts = eachMoon + 1;
+    this.fmAvailableCasts = eachMoon + 1;
```

Comment:
Setting `fmAvailableCasts` in suggestions is fragile as it being called is not guaranteed. A better solution would be to add a getter for `fmAvailableCasts` that calculates it whenever its called.